### ISSUE TITLE
Last file for Autoimmunity

### ISSUE THREAD

**Issue by MaudDLVT** [obophenotype/human-phenotype-ontology#7251]
Hi @pnrobinson and @matentzn ,
Please see the attached file with all the last auto-antibodies.
Many thanks,
Maud
[20211119 Auto immunity File number 4.xlsx](https://github.com/obophenotype/human-phenotype-ontology/files/7710775/20211119.Auto.immunity.File.number.4.xlsx)

**Comment by matentzn** [comment]
I will act on this only after https://github.com/obophenotype/human-phenotype-ontology/issues/7044

In the meantime, @MaudDLVT can you bring the table in the same form as 
https://docs.google.com/spreadsheets/d/1dCU7UCkUp1TLFtsxqq2yXK_AhR5FsPMZtZq3d6TOyTU/edit#gid=0

You can create a new google spreadsheet with me and share it!
Thanks

**Comment by matentzn** [comment]
Ahhh Great! Na, didn't check it, because I just saw Excel :) Looks good! Thanks (sorry about that). I will just wait for @pnrobinson to deal with #7044

**Comment by pnrobinson** [comment]
- [ ] anti-GM1-IgG antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstGM1-IgG PMID: 32460711  ORCID:0000-0002-3387-1836
 - [ ] anti-GD1a-IgG antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstGD1a-IgG PMID: 24829594  ORCID:0000-0002-3387-1836
 - [ ] anti-GM1-IgM antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstGM1-IgM PMID: 34759020  ORCID:0000-0002-3387-1836
 - [ ] anti-GD1a-IgM antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstGD1a-IgM PMID: 31522790  ORCID:0000-0002-3387-1836
 - [ ] anti-GQ1b-IgM antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstGQ1b-IgM PMID: 24829594  ORCID:0000-0002-3387-1836
 - [ ] anti-GD1b-IgM antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstGD1b-IgM PMID: 31837178  ORCID:0000-0002-3387-1836
 - [ ] anti-GT1a-IgM antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstGT1a-IgM PMID: 24829594  ORCID:0000-0002-3387-1836
 - [ ] anti-GQ1b-IgG antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstGQ1b-IgG PMID: 24829594  ORCID:0000-0002-3387-1836
 - [ ] anti-GD1b-IgG antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstGD1b-IgG PMID: 31837178  ORCID:0000-0002-3387-1836
 - [ ] anti-GT1a-IgG antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstGT1a-IgG PMID: 24829594  ORCID:0000-0002-3387-1836
 - [ ] anti-GM2 antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstGM2 PMID: 31522790  ORCID:0000-0002-3387-1836
 - [ ] anti-GM3 antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstGM3 PMID: 34174799  ORCID:0000-0002-3387-1836
 - [ ] anti-GM4 antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstGM4 PMID: 17785314  ORCID:0000-0002-3387-1836
 - [ ] anti-GT1b antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstGT1b PMID: 17785314  ORCID:0000-0002-3387-1836
 - [ ] anti-NF155 antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstNF155 PMID: 34728497, PMID: 32266541 anti-neurofascin-155 antibody positivity ORCID:0000-0002-3387-1836
 - [ ] anti-NF186 antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstNF186 PMID: 33723708, PMID: 32266541 anti-neurofascin 186 antibody positivity ORCID:0000-0002-3387-1836
 - [ ] anti-CNTN1 antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstCNTN1 PMID: 34675937, PMID: 32266541 anti-contactin-1 antibody positivity ORCID:0000-0002-3387-1836
 - [ ] anti-CASPR1 antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstCASPR1 PMID: 32266541 anti-contactin-associated protein 1 antibody positivity ORCID:0000-0002-3387-1836
 - [ ] anti-Hu antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstHu   ORCID:0000-0002-3387-1836
 - [ ] anti-CRMP5 antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstCRMP5   ORCID:0000-0002-3387-1836
 - [ ] anti-Yo antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstYo   ORCID:0000-0002-3387-1836
 - [ ] anti-Ma2 antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstMa2   ORCID:0000-0002-3387-1836
 - [ ] anti-FGFR3 antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstFGFR3   ORCID:0000-0002-3387-1836
 - [ ] anti-AChR antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstAChR PMID: 34228850 anti-acetylcholine receptors antibody positivity ORCID:0000-0002-3387-1836
 - [ ] anti-LRP4 antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstLRP4 PMID: 34233932, PMID: 34228850 anti-lipoprotein receptor-related protein 4 antibody positivity ORCID:0000-0002-3387-1836
 - [ ] anti-MuSK antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstMuSK PMID: 33970337, PMID: 34228850 anti-muscle-specific kinase antibody positivity ORCID:0000-0002-3387-1836
 - [ ] anti-Titin antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstTitin   ORCID:0000-0002-3387-1836
 - [ ] anti-VGCC antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstVGCC   ORCID:0000-0002-3387-1836
 - [ ] anti-Mi2 antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstMi2   ORCID:0000-0002-3387-1836
 - [ ] anti-MDA5 antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstMDA5   ORCID:0000-0002-3387-1836
 - [ ] anti-TIF1-γ antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstTIF1-γ PMID: 27407271 anti-transcription intermediary factor 1-gamma antibody positivity ORCID:0000-0002-3387-1836
 - [ ] anti-SAE antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstSAE PMID: 33343724  ORCID:0000-0002-3387-1836
 - [ ] anti-SAE 2 antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstSAE 2 PMID: 22884621 anti-SUMO-activating enzyme subunit 2 antibody positivity ORCID:0000-0002-3387-1836
 - [ ] anti-SAE 1 antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstSAE 1 PMID: 22884621 anti-SUMO-activating enzyme subunit 1 antibody positivity ORCID:0000-0002-3387-1836
 - [ ] anti-NXP-2 antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstNXP-2 PMID: 33343724 anti-nuclear matrix protein-2 antibody positivity, Anti-MJ/NXP-2 antibody positivity ORCID:0000-0002-3387-1836
 - [ ] anti-SRP antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstSRP PMID: 33343724 anti-signal recognition particle antibody positivity ORCID:0000-0002-3387-1836
 - [ ] anti-HMGCR antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstHMGCR PMID: 26090508 anti-3-hydroxy-3-methylglutaryl-coenzyme A reductase antibody positivity ORCID:0000-0002-3387-1836
 - [ ] anti-PL-7 antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstPL-7 PMID: 33343724 anti-threonyl-tRNA synthetase antibody positivity ORCID:0000-0002-3387-1836
 - [ ] anti-PL-12 antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstPL-12 PMID: 33343724 anti-alanyl-tRNA synthetase antibody positivity ORCID:0000-0002-3387-1836
 - [ ] anti-EJ antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstEJ PMID: 27212602 anti-glycyl tRNA-synthetase antibody positivity ORCID:0000-0002-3387-1836
 - [ ] anti-OJ antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstOJ PMID: 33343724 anti-isoleucyl tRNA-synthetase antibody positivity ORCID:0000-0002-3387-1836
 - [ ] anti-Zo antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstZo PMID: 31665469 anti-phenylalanyl tRNA synthetase antibody positivity, phenylanyl-t-RNA synthetase alpha antibody positivity, anti-Zo alpha antibody positivity ORCID:0000-0002-3387-1836
 - [ ] Anti-Ha antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstHa PMID: 32933078 anti-tyrosyl-tRNA synthetase antibody positivity ORCID:0000-0002-3387-1836
 - [ ] anti-KS antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstKS PMID: 32933078 anti-asparaginyl-tRNA synthetase antibody positivity ORCID:0000-0002-3387-1836
 - [ ] anti-JO1 antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstJO1   ORCID:0000-0002-3387-1836
 - [ ] anti-Ku antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstKu PMID: 12136894, PMID: 24557777  ORCID:0000-0002-3387-1836
 - [ ] anti-Ki antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstKi PMID: 24557777  ORCID:0000-0002-3387-1836
 - [ ] Anti-PM/Scl antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstPM/Scl   ORCID:0000-0002-3387-1836
 - [ ] anti-Ro/SSA antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstRo/SSA   ORCID:0000-0002-3387-1836
 - [ ] anti-Sm antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstSm   ORCID:0000-0002-3387-1836
 - [ ] Anti-U1RNP antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstU1RNP PMID: 10444274 Anti-U1-RNP antibody positivity, Anti-U1-Ribonucleoprotein antibody positivity ORCID:0000-0002-3387-1836
 - [ ] Anti-Ro52 antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstRo52 PMID: 32083547, PMID: 30915082 anti-TRIM21 antibody positivity ORCID:0000-0002-3387-1836
 - [ ] anti-Ro60 antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstRo60 PMID: 32083547, PMID: 30915082 anti-hY-RNA complexes antibody positivity ORCID:0000-0002-3387-1836
 - [ ] Anti-La/SS-B antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstLa/SS-B   ORCID:0000-0002-3387-1836
 - [ ] anti-Scl-70 antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstScl-70 PMID: 34276673 anti-topoisomerase I antibody positivity ORCID:0000-0002-3387-1836
 - [ ] anti-cN1A antibody positivity The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstcN1A PMID: 32933078 anti-cytosolic-5-nucleotidase-1A antibody positivity ORCID:0000-0002-3387-1836

**Comment by MaudDLVT** [comment]
Many thanks @pnrobinson and @matentzn . Could you confirm that the synonyms will be also added? They are quite helpful when searching a specific item.

**Comment by pnrobinson** [comment]
@matentzn adding these by hand because they require additional hierarchical structure.

**Comment by pnrobinson** [comment]
@MaudDLVT I am adding the synonyms. In some cases, the synonyms are just 3 letters and are not standard across the board (i.e., they are only recognizable if you know it is an antibody). I am leaving off these extremely short synonyms since this would be deleterious for the specificity of text mining in general.

**Comment by pnrobinson** [comment]
@MaudDLVT  I am not sure that 
anti-Scl-70 antibody positivity is a synonym for  anti-topoisomerase I antibody positivity 
See -- https://www.sciencedirect.com/topics/medicine-and-dentistry/anti-scl-70-antibodies
See also existing term [Anti-topoisomerase I antibody positivity HP:0030859](https://hpo.jax.org/app/browse/term/HP:0030859)

**Comment by pnrobinson** [comment]
@MaudDLVT 
Anti-ribonucleoprotein antibody positivity HP:0033028
was actually referring to U1. I have updated the label

**Comment by pnrobinson** [comment]
all done!
